crates/pop-parachains/src/new_parachain.rs

```diff
@@ -75,12 +80,72 @@ pub fn instantiate_standard_template(
 	Ok(tag)
 }
 
+/// Create a new tanssi compatible container chain.
+///
+/// # Arguments
+///
+/// * `template` - template to generate the parachain from.
+/// * `target` - location where the parachain will be created.
+/// * `tag_version` - version to use (`None` to use latest).
+fn instantiate_tanssi_template(
+	template: &Parachain,
```

Comment:
Minor, but could alias ContainerChain to Parachain so this reads slightly better.